src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,58 @@
+package task;
+
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Task which has a deadline.
+ *
+ */
+public class Deadline extends Task {
+    private String by;
```

Comment:
I believe `by` can be removed from this class since it is not used any where. Correct me if I am wrong :)